### ISSUE TITLE
itemtest bots models do not walk properly

### ISSUE THREAD

**Issue by joshuadinada** [mastercomfig/team-comtress-2#652]
### Description

in the map `itemtest` if you spawn bots trought the f7 test menu and make them walk around you can notice that they just hover over the ground, not displaying the proper walking animation
### Checklist

- [x] None of the open *or* closed issues document this problem.
- [x] This is a bug, not a new idea.
- [x] This is objectively a bug, and not something that people commonly disagree on.
- [x] This issue occurs on the latest Team Comtress release, or the `master` branch.

- [ ] This issue occurs on the latest vanilla TF2 update.
- [x] This bug is in the code of the game, not in the textures, sounds, or other resources.
- [ ] This issue is listed on the [Source 1 official bugtracker](https://github.com/ValveSoftware/Source-1-Games/issues).
- [ ] This issue occurs with DirectX level 100.
- [x] This issue occurs when using no custom content.
- [x] This issue occurs when using `start_tf2.bat` (or `start_server.bat`) with no custom launch options.
- [ ] This issue occurs when using a listen server (using the `map` command ingame).
- [ ] This issue occurs when using a dedicated server.
- [ ] This bug is in the internal procedures or tooling (e.g. the build process).

### Reproduction

1. open a server with the map `itemtest`
2. join any team and any class
3. press f7, select a class you would like to add and select a weapon for it
4. make the bot walk around, possibly with `bot_mimic`

### Screenshots

https://youtu.be/DESeg8Q03vo

#### Client Settings

- *Server or client:*  ran on a lan server
- *Operating system:*  Windows 10
- *Team Comtress version:*  latest
- *Custom content:*  none
- *Launch options:*  none
#### Driver Version

- *Graphics driver:* 
- *DirectX level:*  98
#### Hardware Specs

- *CPU model:* 
- *GPU model:* 
- *RAM amount:* 
- *TF2 install location:* 
- *TC2 install location:* 
- *TC2 installation method:*

**Comment by mastercoms** [comment]
Does this happen in the version s-1.0.3 ?

**Comment by joshuadinada** [comment]
> Does this happen in the version s-1.0.3 ?

I tested this in the last released build of tc

**Comment by mastercoms** [comment]
Yes but I'm asking for you to test on that version to see if it's a bug with the leaked code

**Comment by joshuadinada** [comment]
> Yes but I'm asking for you to test on that version to see if it's a bug with the leaked code

oh okay sorry

**Comment by joshuadinada** [comment]
> Yes but I'm asking for you to test on that version to see if it's a bug with the leaked code

just tested it, it works on that version

**Comment by mastercoms** [comment]
I've tested, and it is not a regression with any of the patches. It happens with the latest base which I haven't released a `s-` version for yet. Likely to do with the matchmaking emulation I have done for testing, it might be blocking bot movement or something.
